### DISCUSSION TITLE
Error when upgrading deck.gl/* to latest version

### DISCUSSION THREAD

**Discussion by DannyBoris** [visgl/deck.gl#8930] [General]
Hey, i've just upgraded `@deck.gl` to latest version (9.0.14) which causes an internal import issue with `@luma/core`
```
No matching export in 
"node_modules/.pnpm/@luma.gl+core@9.0.0/node_modules/@luma.gl/core/dist/index.js" 
for import "Geometry"'
```
Im using Vue3 as my frontend framework.
I cant find any info about in the issues section so a assume its not a bug but something with my project specifically.
If anyone knows please let me know!

**Comment by ibgreen** [comment]
If your application is using the luma.gl API directly tyou need to look into luma.gl release notes. The luma.gl v9 API has changed considerably, as an example `Geometry` is no longer exported from `@luma.gl/core` but should be imported from `@luma.gl/engine`.

If you are not directly using the luma.gl API in your application, then this is likely a dependency issue in your lock file where some older deck or luma module is still being included.

**Reply by DannyBoris** [reply]
Hey @ibgreen thanks for the reply.
For me is the 2nd scenario - I'm not using luma directly. 
I have removed lock file and node_modules and reinstalled.
Could it be a clash with a optional dependency of v-mapbox? 

<img width="768" alt="Screenshot 2024-06-04 at 14 12 02" src="https://github.com/visgl/deck.gl/assets/42790193/f535154d-64ab-443a-ae27-9d0a1222884d">
<img width="541" alt="Screenshot 2024-06-04 at 14 12 13" src="https://github.com/visgl/deck.gl/assets/42790193/c9497db6-4dad-447d-92e9-2ffb20ca8e43">

I see that this is the only place where the dep is outdated

**Reply by ibgreen** [reply]
Yep, looks like the v-mapbox module (with which I am not familiar) needs to be updated to deck.gl v9.
